### PR TITLE
docs: describe configuration of SSL/TLS

### PR DESCRIPTION
**Pull request by pjenvey**
This is probably enough for #565, for now

r? @jrconlin

### CODE REVIEW

File: docs/running.rst [hunk 1]
```diff
@@ -65,6 +65,39 @@ You may also want to enable human readable logs in the shared config file:
...
+-------------------
+
+The Push protocol mandates use of SSL/TLS connections to ensure the integrity
+of push subscription data. This rule can be ignored for testing purposes
+only. This requires a valid, trusted TLS certificate (for example, from `Let's
+Encrypt`_).
+
+The ``ssl_key`` and ``ssl_cert`` options configure a TLS certificate for both
+the endpoint's HTTPS and connection node's WebSocket Secure (WSS) connections:
+
```
Comment (jrconlin) [inline docs/running.rst]: Wonder if we should note that different creds can be used for endpoint and connection handlers by moving these option definitions to the appropriate config files? (Could be consideration if you don't have a wild card cert.)

File: docs/running.rst [hunk 2]
```diff
@@ -65,6 +65,39 @@ You may also want to enable human readable logs in the shared config file:
...
+    ; TLS private key and certificate file paths, used for the main
+    ; connection and endpoint listeners. If omitted, TLS will not be
+    ; used.
+    ssl_key = /etc/letsencrypt/live/EXAMPLE.COM/privkey.pem
+    ssl_cert = /etc/letsencrypt/live/EXAMPLE.COM/cert.pem
+    ; Path to file containing any additional parameters for the Ephemeral
+    ; Diffie-Hellman
+    #ssl_dh_param =
+
+The connection between the endpoint and connection nodes should not be exposed
```
Comment (jrconlin) [inline docs/running.rst]: "exposed" or "accessible"? Really, it should be locked to only other push server machines, but that might be more complicated than necessary.

File: docs/running.rst (hunk 1 above)
Comment (pjenvey) [inline docs/running.rst]: Sounds good

**Comment by jrconlin** [comment]
r+

**Pull request merged**